Fails to compile due to woff2

See https://github.com/google/woff2/pull/176
```
  cargo:rerun-if-env-changed=CXXFLAGS_aarch64-unknown-linux-gnu
  CXXFLAGS_aarch64-unknown-linux-gnu = None
  CARGO_ENCODED_RUSTFLAGS = Some(-Clink-arg=--ld-path=wild)
  cargo:warning=In file included from src/deps/woff2/src/woff2_out.cc:9:
  cargo:warning=src/deps/woff2/include/woff2/output.h:73:25: error: expected ')'
  cargo:warning=   73 |   WOFF2MemoryOut(uint8_t* buf, size_t buf_size);
  cargo:warning=      |                 ~       ^
  cargo:warning=      |                         )
  cargo:warning=src/deps/woff2/include/woff2/output.h:79:3: error: 'uint8_t' doe
  cargo:warning=   79 |   uint8_t* buf_;
  cargo:warning=      |   ^~~~~~~
  cargo:warning=src/deps/woff2/include/woff2/output.h:16:1: note: 'uint8_t' is d
  cargo:warning=   15 | #include <string>
  cargo:warning=  +++ |+#include <cstdint>
  cargo:warning=   16 |
  cargo:warning=src/deps/woff2/src/woff2_out.cc:48:31: error: expected construct
  cargo:warning=   48 | WOFF2MemoryOut::WOFF2MemoryOut(uint8_t* buf, size_t buf_size)
  cargo:warning=      |                               ^
  cargo:warning=src/deps/woff2/src/woff2_out.cc: In member function 'virtual bool woff2::WOFF2MemoryOut::Write(const void*, size_t, size_t)':
  cargo:warning=src/deps/woff2/src/woff2_out.cc:61:15: error: 'buf_' was not declared in this scope; did you mean 'buf'?
  cargo:warning=   61 |   std::memcpy(buf_ + offset, buf, n);
  cargo:warning=      |               ^~~~
  cargo:warning=      |               buf
  exit status: 0
  exit status: 0
  exit status: 0
  exit status: 1
  cargo:warning=ToolExecError: command did not execute successfully (status code exit status: 1): LC_ALL="C" "sccache" "c++" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-gdwarf-4" "-fno-omit-frame-pointer" "-std=c++11" "-I" "src/deps/woff2/include" "-I" "src/deps/brotli/c/include" "-o" "/home/theo/src/servo/target/debug/build/fontsan-137ea396f2ffc2fa/out/6f47f7afeb56ca5a-woff2_out.o" "-c" "src/deps/woff2/src/woff2_out.cc"
  exit status: 0
```

--- a/src/deps/woff2/include/woff2/output.h
+++ b/src/deps/woff2/include/woff2/output.h
@@ -9,6 +9,8 @@
 #ifndef WOFF2_WOFF2_OUT_H_
 #define WOFF2_WOFF2_OUT_H_
 
+#include <stdint.h>
+
 #include <algorithm>
 #include <cstring>
 #include <memory>

